tests/behat/behat-lando.yml

```diff
@@ -0,0 +1,24 @@
+imports:
+  - behat-pantheon.yml
+
+default:
+  suites:
+    default:
+      paths:
+        - ./tests/behat
```

Comment:
Why are the `paths` different for `behat-lando.yml` and `behat-pantheon.yml` when they are in the same directory?

Reply:
@stevector I will update. When running locally using `./` ran all the tests for Composer directories since the Behat command is ran from the git root. I suspect it doesn't do this on CircleCI as Behat tests run in isolation, without dependencies installed.